addons/im_livechat/tests/test_message.py

```diff
@@ -81,7 +81,6 @@ def test_message_format(self):
             'date': message.date,
             'write_date': message.write_date,
             'create_date': message.create_date,
-            'history_partner_ids': [],
```

Comment:
We probably want to test the full format rather than removing keys here

Reply:
Works for me